sjsonnet/test/src/sjsonnet/ParserTests.scala

```diff
@@ -13,13 +13,13 @@ object ParserTests extends TestSuite{
       parse("true") ==> True(pos(0))
 
       parse("123 + 456 + 789") ==>
-        BinaryOp(pos(10), BinaryOp(pos(4), Num(pos(0), 123), BinaryOp.`+`, Num(pos(6), 456)), BinaryOp.`+`, Num(pos(12), 789))
+        BinaryOp(pos(10), BinaryOp(pos(4), Num(pos(0), 123), BinaryOp.OP_+, Num(pos(6), 456)), BinaryOp.OP_+, Num(pos(12), 789))
 
       parse("1 * 2 + 3") ==>
-        BinaryOp(pos(6), BinaryOp(pos(2), Num(pos(0), 1), BinaryOp.`*`, Num(pos(4), 2)), BinaryOp.`+`, Num(pos(8), 3))
+        BinaryOp(pos(6), BinaryOp(pos(2), Num(pos(0), 1), BinaryOp.OP_*, Num(pos(4), 2)), BinaryOp.OP_+, Num(pos(8), 3))
 
       parse("1 + 2 * 3") ==>
-        BinaryOp(pos(2), Num(pos(0), 1), BinaryOp.`+`, BinaryOp(pos(6), Num(pos(4), 2), BinaryOp.`*`, Num(pos(8), 3)))
+        BinaryOp(pos(2), Num(pos(0), 1), BinaryOp.OP_+, BinaryOp(pos(6), Num(pos(4), 2), BinaryOp.OP_*, Num(pos(8), 3)))
     }
     test("duplicateFields") {
       parseErr("{ a: 1, a: 2 }") ==> """Expected no duplicate field: a:1:14, found "}""""
```

Comment:
Can we add a few tests to `ParserTests` to validate the construction of static `Val.Arr`s and `Val.Obj`s? A lot of the new changes are semantically indistinguishable from the status quo, and thus wouldn't get validated through the normal course of compiling jsonnet. We're also starting to have edge cases that are worth validating in tests, e.g. nested static arrays, nested static objects, alternating nested static arrays and static objects